exchanges/kucoin/kucoin_wrapper.go

```diff
@@ -2009,3 +2010,57 @@ func (ku *Kucoin) GetOpenInterest(ctx context.Context, k ...key.PairAsset) ([]fu
 	}
 	return resp, nil
 }
+
+// SynchroniseFees updates the fee schedule for the exchange
+func (ku *Kucoin) SynchroniseFees(ctx context.Context, a asset.Item) error {
+	const defaultFeeRequestWindow = 10
+
+	if !ku.SupportsAsset(a) {
+		return fmt.Errorf("%w %v", asset.ErrNotSupported, a)
+	}
+
+	if a != asset.Spot {
+		return common.ErrNotYetImplemented
```

Comment:
I actually can't find a futures end point for this. If there is one though, I think it should be implemented